### PR TITLE
Adding an .about.yml and LICENSE.md

### PR DESCRIPTION
**Pull request by DavidEBest**

### CODE REVIEW

File: .about.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+stage: discovery
+
+# Description of the project
+description: Toward a more secure (and automated) future.
+
+# Tags that describe the project or aspects of the project
+tags:
+- ato
+  compliance
+  testing
```
Comment (afeld) [inline .about.yml]: I think each line here needs dashes.

Comment (afeld) [inline .about.yml]: I'd also add "security".

File: .about.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+#- 
+
+# Brief descriptions of significant project developments
+milestones:
+- Project began in January 2016 
+
+# Technologies used to build the project
+stack:
+- Jekyll
+  Concourse.CI
```
Comment (afeld) [inline .about.yml]: I think this line needs a dash at the beginning. I'd also include OWASP ZAP.

**Comment by DavidEBest** [comment]
Updated!

**Pull request merged**